src/modules/Microsoft.Network/privateEndpoints/.bicep/nested_roleAssignments.bicep

```diff
@@ -0,0 +1,93 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
```

Comment:
Can you remove the .bicep and .test folders? I don't think we need those.

Reply:
We do need the .bicep folder, it contains resources referenced in the main module